apps/condo/domains/organization/integrations/sbbol/utils/getSbbolUserInfoErrors.spec.js

```diff
@@ -21,9 +21,36 @@ const DEFAULT_SEED = {
 }
 
 describe('getSbbolUserInfoErrors', () => {
-    test('no userGuid', () => {
-        const errors = getSbbolUserInfoErrors(DEFAULT_SEED)
-        expect(errors).toEqual(['must have required property \'userGuid\''])
+    test('no userGuid and no sub', () => {
+        const { sub, userGuid, ...info } = {
+            ...DEFAULT_SEED,
+            userGuid: '5a99ca6e22f321af3a',
```

Comment:
Why do we set `userGuid` if we don't use it later?

Reply:
fixed